.deploy.yml

```diff
@@ -56,14 +56,18 @@ jobs:
   - powershell: ./impact.ps1
     condition: eq(variables['Build.SourceBranch'], 'refs/heads/v3')
     env:
-      OP_BUILD_USER_TOKEN: $(OpBuildUserToken)
```

Comment:
this one is used for running config migration in edge-developer

Reply:
@yufeih  is saying that you don't need it anymore?